english/God's-own-country/kerala-story.md

```diff
@@ -0,0 +1,4 @@
+Kerala, a state on India's tropical Malabar Coast, has nearly 600km of Arabian Sea shoreline.
+It's known for its palm-lined [beaches](../beaches/kerala2.md) and its [backwaters](../lagoons/kerala3.md), [rivers](../44rivers/kerala1.md) a network of canals popular for cruises. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.